src/server.c

```diff
@@ -1634,7 +1710,9 @@ qd_listener_t *qd_server_listen(qd_dispatch_t *qd, const qd_server_config_t *con
     li->server      = qd_server;
     li->config      = config;
     li->context     = context;
+    li->log_message = populate_log_message(config);
```

Comment:
It's confusing that "log_message" sometimes means a character string and sometimes means a bit-mask.